app/views/components/docs/share-links.yml

```diff
@@ -0,0 +1,23 @@
+name: Share links
+description: Social media links, linking to facebook and twitter share pages
```

Comment:
I think it would be worth adding a body here to explain a few things:

- the tracking
- the js-share-facebook/twitter classes on the elements (presumably linked to the tracking?)
- that the URL passed must be a complete share URL, i.e. the component does not automatically take a URL and process it into a link that when clicked will share something on Facebook